src/map/clif.cpp

```diff
@@ -22843,10 +22845,17 @@ bool clif_parse_stylist_buy_sub( map_session_data* sd, _look look, int16 index )
 	}
 
 	switch( look ){
+		case LOOK_BODY2:
```

Comment:
This is updating or sending logic.
Validating the requirements should be done beforehand (and not after paying zeny and deleting items).